cmd/tlsconfig/tlsconfig.go

```diff
@@ -124,8 +124,6 @@ func mapTLSVersions(tlsVersions []string) []int {
 			versions = append(versions, tls.VersionTLS11)
 		case "TLSv1":
 			versions = append(versions, tls.VersionTLS10)
-		case "SSLv3":
-			versions = append(versions, tls.VersionSSL30)
```

Comment:
This is used to map the TLS version published by Mozilla (https://statics.tls.security.mozilla.org/server-side-tls-conf.json) to Go version  when generating the TLS rule. 

We cannot directly remove it since we support 2 major older Go versions from latest.  Most likely, we have to make it conditional with a build tag  when it is entirely removed form newer Go versions.

Reply:
I reworked the PR to use build tags, with  some comments in the source to help the future know when to discard the older of the `mapTLSVersions()`.

Reply:
Looks good! Thanks!